### PR TITLE
Register oc metrics

### PR DESCRIPTION
**Pull request by lanzafame**
This PR exposes OpenCensus metrics from go-ipfs to enable the collection and export of QUIC metrics. This was done on a separate port to the prometheus metrics as there were issues getting OC metrics into the prometheus registry. I have gotten it to work the other way but I can't seem to get to work this way. 

There is also a small fix to the plugin loading script.

### CODE REVIEW

**Comment by lanzafame** [comment]
@marten-seemann Ping me when you have merged the metrics package and tagged into quic-go and I will update this PR.

**Comment by lanzafame** [comment]
@marten-seemann any update on tagging quic-go?

**Comment by marten-seemann** [comment]
@lanzafame The new quic-go release now has metrics support. All that's left to do is to enable metrics collection in go-libp2p-quic-transport: https://github.com/libp2p/go-libp2p-quic-transport/pull/172.

**Comment by marten-seemann** [comment]
Hi @lanzafame, I just released a v0.8.1 of go-libp2p-quic-transport that enables metrics collection (but doesn't pull in the recent stream interface changes yet). Could you update this PR?

**Comment by lanzafame** [comment]
@marten-seemann @aschmahmann This is now up-to-date and working.
Screen Shot 2020-09-29 at 09 46 13

**Comment by marten-seemann** [comment]
Once #7769 is merged, this PR should be ready to go (minus the quic-go update to go.mod).

**Review by aschmahmann — APPROVED** [review]

**Comment by aschmahmann** [comment]
@lanzafame @marten-seemann I merged this since it looks like it was just blocked on a quic-go update which is now in. I noticed that the log level of the opencensus initialization was Error which is pretty loud so I decreased it to Info in https://github.com/ipfs/go-ipfs/pull/7815.

If there's anything I've missed here lmk and we can get it into the next RC.

**Pull request merged**
